.github/workflows/ci.yml

```diff
@@ -19,41 +19,18 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - uses: pnpm/action-setup@v2
+
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+          cache: pnpm
 
       - name: Install deps
-        run: npm ci
+        run: pnpm i --frozen-lockfile
 
       - name: Build and Test
         run: npm run test
         env:
           VITE_MOCK_USER_ADDRESS: ${{ secrets.VITE_MOCK_USER_ADDRESS }}
           VITE_MOCK_USER_PRIVATE_KEY: ${{ secrets.VITE_MOCK_USER_PRIVATE_KEY }}
-
-  publish:
-    name: npm-publish
-    runs-on: ubuntu-latest
-    needs: ['test']
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 18
-      - run: npm install
-      - id: publish
-        uses: JS-DevTools/npm-publish@v1
-        with:
-          token: ${{ secrets.NPM_TOKEN }}
-      - name: Create Release
```

Comment:
We need this?

Reply:
I moved it to `.github/workflows/release.yml`
We should publish only when release, not every PR or commits.

Reply:
We don't publish after release but instead work in reverse. The current workflow is that we bump the "version" manually in package.json and push it to main branch, which triggers the `JS-DevTools/npm-publish@v1` to check and publish, then the `actions/create-release@v1` to create a GitHub release, then the "release.yml" file gets workinh to run `changelogithub` to update the Release content.

Reply:
Added `release` script. It's actually the same thing. The ci workflow after changed is 
- `pnpm release` - it will bump a new version and commit & tag & push
- after pushing, it will trigger CI `Release` 
- CI will publish to npm registry and create a GitHub release.

Reply:
> and create a GitHub release.

I think we still need `actions/create-release@v1` to create it?